Orange/widgets/data/owpreprocess.py

```diff
@@ -29,7 +29,7 @@
 from Orange.widgets.utils.sql import check_sql_input
 from Orange.widgets.utils.widgetpreview import WidgetPreview
 from Orange.widgets.widget import Input, Output
-
+from Orange.preprocess import Normalize
```

Comment:
You can use `Normalize`, but since the code currently uses `Scale`, why wouldn't you modify `Scale` to support `zero_based` flag (which would default to `True`, but the widget would set it to `False` for `NormalizeSpan2`? This could simplify `createInstance` - you'd just use two lists or dictionaries instead of all the ifs.